python/dask_cudf/dask_cudf/backends.py

```diff
@@ -695,15 +700,142 @@ def from_dict(
         )
 
     @staticmethod
-    def read_parquet(*args, engine=None, **kwargs):
+    def read_parquet(path, filesystem="fsspec", engine=None, **kwargs):
```

Comment:
This is an API break because the *args previously consumed all non-keyword arguments. We need to keep *args here, and we can deprecate / etc. later on.

Reply:
Good call - We can keep the *args around for now.